scripts/build-appimage.sh

```
@@ -86,9 +87,10 @@ if [ -n "$QT_PLUGIN_PATH" ] && [ -d "$QT_PLUGIN_PATH/multimedia" ]; then
   rm -f "$QT_PLUGIN_PATH/multimedia/libgstreamermediaplugin.so"
 fi
 pushd $INSTALLER_FOLDER
-VERSION=$VERSION $LINUXDEPLOY --appdir $DEPLOY_FOLDER \
+OUTPUT="$INSTALLER_FOLDER/Moonlight-VPlus-$VERSION-$APPIMAGE_ARCH.AppImage" \
```

Shellcheck (0.11.0)

[warning] This expansion will not see the mentioned assignment.

(SC2098)